package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "zapier-platform-core",
-  "version": "0.9.12",
-  "description": "The core SDK for the Zapier platform.",
+  "version": "0.10.1",
```

Comment:
Careful, when running `npm version patch` (like we should be doing) it will increment this - which will push to `0.10.2`. Looks like `0.9.12` missed the bump to `0.10.0`.